kubernetes/kubernetes.go

```diff
@@ -387,6 +388,10 @@ func (k KubernetesJobClient) Create(jobSpec *batchv1.JobSpec) (*batchv1.Job, err
 }
 
 func (k KubernetesJobClient) SetJobSchedule(schedule CronSchedule, jobSpec *batchv1.JobSpec) error {
+	successfulJobsHistoryLimit := helpers.GetEnvInt32("SUCCESSFUL_JOBS_HISTORY_LIMIT", 2)
+	failedJobsHistoryLimit := helpers.GetEnvInt32("FAILED_JOBS_HISTORY_LIMIT", 1)
+	concurrencyPolicy := batchv1.ConcurrencyPolicy(helpers.GetEnv("CONCURRENCY_POLICY", "Forbid"))
```

Comment:
Options we have


	// Specifies how to treat concurrent executions of a Job.
	// Valid values are:
	// - "Allow" (default): allows CronJobs to run concurrently;
	// - "Forbid": forbids concurrent runs, skipping next run if previous run hasn't finished yet;
	// - "Replace": cancels currently running job and replaces it with a new one
	// +optional
	
The forbid might be useful if you schedule a job every minute and it does a shit ton of compute making it take 4eva.  With the current default, you have a ton queuing up and it could OOM your cluster. I'm adding it as an env var so we can config it for local tests